.github/workflows/test-node.yaml

```diff
@@ -20,5 +20,5 @@ jobs:
 
       - name: Run Node test
         run: |
-          npm install --global prettier@2.2.0
+          npm install --global prettier@2.3.2
```

Comment:
Can we try to take version from package.json ?
For example: `(cd pkg/new-ui/v1beta1/frontend && npm install prettier)`
I think it will be easier to maintain.
WDYT @seong7 ?

Reply:
If this will not work (with `--global` flag I unable to install specific version from `package.json`), try to use this command:
```
version=$(cd pkg/new-ui/v1beta1/frontend && node -p "require('prettier/package.json').version")
npm install --global prettier@${version}
```